src/main/java/org/broadinstitute/hellbender/tools/pon/coverage/pca/PCATangentNormalizationUtils.java

```diff
@@ -0,0 +1,305 @@
+     *   </ul>
+     *
+     * @param pon -- never {@code null}
+     * @param profile -- never {@code null}.  Must contain data for at least one sample.
+     * @param ctx spark context.  Use {@code null} if no context is available
+     * @param doFactorNormalization if true, perform factor normalization (set to false for normalizing normals in a PoN that have already been factor normalized)
+     * @return never {@code null}
+     */
+    static PCATangentNormalizationResult tangentNormalize(final PCACoveragePoN pon, final ReadCountCollection profile, final boolean doFactorNormalization, final JavaSparkContext ctx) {
+        Utils.nonNull(pon, "PoN cannot be null.");
+        Utils.nonNull(profile, "Proportional coverages cannot be null.");
+        ParamUtils.isPositive(profile.columnNames().size(), "Column names cannot be an empty list.");
+
+        //normals stored in a PoN may already be factor normalized
+        final ReadCountCollection factorNormalizedCoverage = doFactorNormalization ? mapTargetsToPoNAndFactorNormalize(profile, pon) : profile;
+
+        return tangentNormalize(factorNormalizedCoverage, pon.getPanelTargetNames(), pon.getReducedPanelCounts(), pon.getReducedPanelPInverseCounts(), ctx);
+    }
+
+    /**
+     * Project all of the normals used to create the PoN into the reduced panel using the raw PoN data.
+     * This is required to calculate target variances for PoN initialization
+     * (since target variances are considered part of {@link PCACoveragePoN}, this cannot be done simply using
+     * {@link CoveragePanelOfNormals#normalizeNormalsInPoN} since the PoN is not fully initialized).
+     */
+    static PCATangentNormalizationResult tangentNormalizeNormalsInPoN(final ReadCountCollection normalizedCounts,
+                                                                   final List<String> panelTargetNames,
+                                                                   final RealMatrix reducedPanelCounts,
+                                                                   final RealMatrix reducedPanelPInvCounts,
+                                                                   final JavaSparkContext ctx) {
+        Utils.nonNull(normalizedCounts, "Normalized counts cannot be null.");
+        Utils.nonNull(panelTargetNames, "Panel target names cannot be null.");
+        Utils.nonNull(reducedPanelCounts, "Reduced-panel counts cannot be null.");
+        Utils.nonNull(reducedPanelPInvCounts, "Reduced-panel pseudoinverse cannot be null.");
+        // For each sample in the PoN, tangent normalize against the qc reduced PoN.
+        logger.info("Tangent normalizing the normals (normalized by target factors) ...");
+        return tangentNormalize(normalizedCounts, panelTargetNames, reducedPanelCounts, reducedPanelPInvCounts, ctx);
+    }
+
+    /**
+     * Returns a target-factor--normalized {@link ReadCountCollection} given a {@link PCACoveragePoN}..
```

Comment:
done.  i'm not happy with the use of both "target factor normalized" and "factor normalized" along with inconsistent hyphenation, but i guess there are worse things